benchmark_runner/common/oc/oc.py

```diff
@@ -1356,8 +1376,12 @@ def get_vm_access(self, vm_name: str = '', namespace: str = environment_variable
         :param namespace:
         :return: virtctl_status 'True' if successful, or an error message if it fails.
         """
-        virtctl_vm_cmd = f"virtctl ssh --local-ssh=true --local-ssh-opts='-o BatchMode=yes' --local-ssh-opts='-o PasswordAuthentication=no' --local-ssh-opts='-o ConnectTimeout=2' root@{vm_name} -n {namespace}"
-        check_virtctl_vm_cmd = f"virtctl ssh --local-ssh=true --local-ssh-opts='-o BatchMode=yes' --local-ssh-opts='-o PasswordAuthentication=no' --local-ssh-opts='-o ConnectTimeout=2' root@{vm_name} -n {namespace} 2>&1 |egrep 'denied|verification failed'  && echo 'True' || echo 'False'"
+        if self.is_virtctl_ge(target_version="1.6.0"):
+            virtctl_vm_cmd = f"virtctl ssh --local-ssh=true --local-ssh-opts='-o BatchMode=yes' --local-ssh-opts='-o PasswordAuthentication=no' --local-ssh-opts='-o ConnectTimeout=2' root@vm/{vm_name}/{namespace}"
+            check_virtctl_vm_cmd = f"virtctl ssh --local-ssh=true --local-ssh-opts='-o BatchMode=yes' --local-ssh-opts='-o PasswordAuthentication=no' --local-ssh-opts='-o ConnectTimeout=2' root@vm/{vm_name}/{namespace} 2>&1 |egrep 'denied|verification failed'  && echo 'True' || echo 'False'"
```

Comment:
A lot of duplicated code here.  You could have a single method `build_base_virtctl_ssh_command` and then add to it whatever is needed for the particular use case.